lib/retriable/exponential_backoff.rb

```diff
@@ -39,7 +39,7 @@ def randomize(interval)
       delta = rand_factor * interval * 1.0
       min = interval - delta
       max = interval + delta
-      rand(min..max)
+      Random.new(42).rand(min..max)
```

Comment:
Passing in `42` actually makes the seed number static doesn't it? This means that for a given min and max, it'll always return the same "random" return value no?